bin/setup

```diff
@@ -22,10 +22,9 @@ chdir APP_ROOT do
   # Install JavaScript dependencies if using Yarn
   # system('bin/yarn')
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
+  puts "\n== Copying sample files =="
+  cp 'config/database.yml.example', 'config/database.yml' unless File.exist?('config/database.yml')
+  cp 'config/bunny.yml.example', 'config/bunny.yml' unless File.exist?('config/bunny.yml')
 
```

Comment:
Just means bin/setup gets everything up and running to dive straight in and run a server.